script.js

```diff
@@ -0,0 +1,171 @@
+const toDoForm = document.querySelector(".toDo__form"),
+  toDoInput = document.querySelector("input"),
+  toDoWaitingItems = document.querySelector(".waiting__items"),
+  toDoFinishedItems = document.querySelector(".finished__items"),
+  toDoInputIcon = document.querySelector(".input__icon"),
+  waitingListCount = document.querySelector(".waiting__number"),
+  finishedListCount = document.querySelector(".finished__number");
+
+const TODO_LS = "toDos";
```

Comment:
localStorage 이름같이 절대 바뀌지 않는 상수는 미리 선언해 놓는 거 너무 좋아요! 
다만, 저도 시원님께 코멘트를 받았는데 상수명이 무엇을 나타내는지 제대로 쓰지 않으면 나중에 헤갈릴거 같더라구요!!
```suggestion
const LOCALSTORAGE_KEY = "toDos";
```

Reply:
앗 그렇네요..! 프로그램이 조금만 더 커지면 확실히 상수명이 헷갈릴 것 같아요..! 공유해주셔서 감사합니다!